commit 34fecb92ca493b31699902e098fc64942bc93b56 (refs/changes/59/2059/1)
Author: Josh Wilsdon <jwilsdon@joyent.com>
Date:   2017-06-08T12:39:47-07:00 (2 years, 4 months ago)
    
    OS-6171 when using rsync in vmadm we should workaround hostile GZ setups

diff --git a/src/vm/node_modules/VM.js b/src/vm/node_modules/VM.js
index 3ce23319..d4c023f4 100644
--- a/src/vm/node_modules/VM.js
+++ b/src/vm/node_modules/VM.js
@@ -2441,7 +2441,7 @@ function copyFilesystemData(payload, filesystem, log, callback)
 
         // Need to add the trailing '/' in so we copy only the *contents* of the
         // dir instead of the dir itself.
-        args = ['-aq', existing + '/', filesystem.source];
+        args = ['--numeric-ids', '-aq', existing + '/', filesystem.source];
         log.info({
             cmd: cmd,
             args: args,
